content/en/docs/components/model-registry/installation.md

```diff
@@ -18,26 +18,35 @@ These are the minimal requirements to install Model Registry:
 
 ## Installing Model Registry
 
-You can skip this step if you have already installed Kubeflow >=1.9. Your Kubeflow
-deployment includes Model Registry ([see tracker issue](https://github.com/kubeflow/manifests/issues/2631)).
+Kubeflow Model registry may be installed as part of a Kubeflow Platform, or as a standalone component.
+The best option for you will depend on your specific requirements.
 
-To install Model Registry as part of Kubeflow, follow the
-[Kubeflow installation guide](/docs/started/installing-kubeflow/).
+### Installing on Kubeflow Platform
+
+Kubeflow Model Registry is available as an opt-in alpha component in Kubeflow Platform 1.9+, see [Installing Kubeflow](/docs/started/installing-kubeflow/) to learn more about deploying the Kubeflow Platform.
+
+If you have deployed the raw manifests, you may follow [these instructions](https://github.com/kubeflow/manifests/tree/master/apps/model-registry/upstream#readme) to deploy Model Registry; please raise any feedback on [`kubeflow/manifests#2631`](https://github.com/kubeflow/manifests/issues/2631).
```

Comment:
Do you want to create an issue in the model-registry repo, not in the manifests ?
So it will be easier for your to track and triage it ?

Reply:
oh I see, I wanted them to refer to the original issue, but you are probably right they might go off and create new one (but in that case they might be using the "wrong" repo). 
amended with 276c49eb9e954d960e2c256aaf019f5827456ade